### PR TITLE
Use the "recordified" MPL library.

### PR DESCRIPTION
**Pull request by jakaarl**

### CODE REVIEW

File: trade-module/src/main/java/io/edpn/backend/trade/application/service/RequestStationArrivalDistanceService.java [hunk 1]
```diff
@@ -26,10 +26,9 @@ public boolean isApplicable(Station station) {
 
     @Override
     public void request(Station station) {
-        StationDataRequest stationDataRequest = new StationDataRequest();
-        stationDataRequest.setStationName(station.getName());
-        stationDataRequest.setSystemName(station.getSystem().getName());
-
+        StationDataRequest stationDataRequest = new StationDataRequest(
+                null, station.getName(), station.getSystem().getName()
```
Comment (pveeckhout) [inline trade-module/src/main/java/io/edpn/backend/trade/application/service/RequestStationArrivalDistanceService.java]: requesting module is empty, is this set somewhere else?

ohterwise this must be "trade" here

File: trade-module/src/main/java/io/edpn/backend/trade/application/service/RequestStationLandingPadSizeService.java [hunk 2]
```diff
@@ -27,10 +27,9 @@ public boolean isApplicable(Station station) {
 
     @Override
     public void request(Station station) {
-        StationDataRequest stationDataRequest = new StationDataRequest();
-        stationDataRequest.setStationName(station.getName());
-        stationDataRequest.setSystemName(station.getSystem().getName());
-
+        StationDataRequest stationDataRequest = new StationDataRequest(
+                null, station.getName(), station.getSystem().getName()
```
Comment (pveeckhout) [inline trade-module/src/main/java/io/edpn/backend/trade/application/service/RequestStationLandingPadSizeService.java]: requesting module is empty, is this set somewhere else?

ohterwise this must be "trade" here

File: trade-module/src/main/java/io/edpn/backend/trade/application/service/RequestStationPlanetaryService.java [hunk 3]
```diff
@@ -26,10 +26,9 @@ public boolean isApplicable(Station station) {
 
     @Override
     public void request(Station station) {
-        StationDataRequest stationDataRequest = new StationDataRequest();
-        stationDataRequest.setStationName(station.getName());
-        stationDataRequest.setSystemName(station.getSystem().getName());
-
+        StationDataRequest stationDataRequest = new StationDataRequest(
+                null, station.getName(), station.getSystem().getName()
```
Comment (pveeckhout) [inline trade-module/src/main/java/io/edpn/backend/trade/application/service/RequestStationPlanetaryService.java]: requesting module is empty, is this set somewhere else?

ohterwise this must be "trade" here

File: trade-module/src/main/java/io/edpn/backend/trade/application/service/RequestStationRequireOdysseyService.java [hunk 4]
```diff
@@ -26,10 +26,9 @@ public boolean isApplicable(Station station) {
 
     @Override
     public void request(Station station) {
-        StationDataRequest stationDataRequest = new StationDataRequest();
-        stationDataRequest.setStationName(station.getName());
-        stationDataRequest.setSystemName(station.getSystem().getName());
-
+        StationDataRequest stationDataRequest = new StationDataRequest(
+                null, station.getName(), station.getSystem().getName()
```
Comment (pveeckhout) [inline trade-module/src/main/java/io/edpn/backend/trade/application/service/RequestStationRequireOdysseyService.java]: requesting module is empty, is this set somewhere else?

ohterwise this must be "trade" here

File: trade-module/src/main/java/io/edpn/backend/trade/application/service/RequestSystemCoordinatesService.java [hunk 5]
```diff
@@ -26,9 +26,9 @@ public boolean isApplicable(System system) {
 
     @Override
     public void request(System system) {
-        SystemDataRequest stationDataRequest = new SystemDataRequest();
-        stationDataRequest.setSystemName(system.getName());
-
+        SystemDataRequest stationDataRequest = new SystemDataRequest(
+                null, system.getName()
```
Comment (pveeckhout) [inline trade-module/src/main/java/io/edpn/backend/trade/application/service/RequestSystemCoordinatesService.java]: requesting module is empty, is this set somewhere else?

ohterwise this must be "trade" here

File: trade-module/src/main/java/io/edpn/backend/trade/application/service/RequestSystemEliteIdService.java [hunk 6]
```diff
@@ -26,9 +26,9 @@ public boolean isApplicable(System system) {
 
     @Override
     public void request(System system) {
-        SystemDataRequest systemDataRequest = new SystemDataRequest();
-        systemDataRequest.setSystemName(system.getName());
-
+        SystemDataRequest systemDataRequest = new SystemDataRequest(
+                null, system.getName()
```
Comment (pveeckhout) [inline trade-module/src/main/java/io/edpn/backend/trade/application/service/RequestSystemEliteIdService.java]: requesting module is empty, is this set somewhere else?

ohterwise this must be "trade" here

**Review by pveeckhout — CHANGES_REQUESTED** [review]

File: trade-module/src/main/java/io/edpn/backend/trade/application/service/RequestStationArrivalDistanceService.java (hunk 1 above)
Comment (jakaarl) [reply trade-module/src/main/java/io/edpn/backend/trade/application/service/RequestStationArrivalDistanceService.java]: I was merely keeping things "functionally unchanged" - the old code didn't set requesting module. I'll look into changing it to an enum (ed-pilots-network/backend-util#3) and passing it to the constructors.

**Review by pveeckhout — APPROVED** [review]

File: build.gradle [hunk 7]
```diff
@@ -79,9 +79,9 @@ subprojects {
         kafkaVersion = '3.0.8'
         springdocOpenapiVersion = '2.1.0'
         prometheusVersion = '1.11.1'
-        backendUtilVersion = '0.0.1-SNAPSHOT'
+        backendUtilVersion = '0.0.2-20230806.194019-5'
```
Comment (officialyinsane) [inline build.gradle]: Changed to unique snapshot versioning, and this doesn't match parent version?

**Review by officialyinsane — APPROVED** [review]

**Pull request merged**